### PR TITLE
[Chat][Feature] Disable send button when message is empty 

### PR DESCRIPTION
**Pull request by ShaunaSong**
## Purpose

* Cannot send the empty messages
* Delete message input hint 

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

## Pull Request Type
What kind of change does this Pull Request introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Pull Request Checklist

- [ ] Public API changes
- [ ] Verified for cross-platform
- [ ] Synced with iOS, Web team
- [ ] Internal review completed
- [ ] UI Changes
  - [ ] Screen captures included
  - [ ] Tested for Light/Dark mode
  - [ ] Tested for screen rotation
  - [ ] Tested on Tablet and small screen device (5")
  - [ ] Include localization changes
- [ ] Tests
  - [ ] Memory leak analysis performed
  - [ ] Tested on API 21, 26 and latest 
  - [ ] Tested for background mode
  - [ ] Unit Tests Included
  - [ ] UI Automated Tests Included

## How to Test

* Open...
* Click on...

## What to Check
Verify that the following are valid
* ...

### CODE REVIEW

File: azure-communication-ui/chat/src/main/res/values/azure_communication_ui_chat_strings.xml [hunk 1]
```diff
@@ -6,7 +6,6 @@
     <string name="azure_communication_ui_chat_chat_action_bar_title">Chat</string>
     <string name="azure_communication_ui_chat_title_activity_compose_chat">ComposeChatActivity</string>
     <string name="azure_communication_ui_chat_demo_app_title">Chat Composite Demo App</string>
-    <string name="azure_communication_ui_chat_enter_a_message">Hey, whats up?</string>
```
Comment (iaulakh) [inline azure-communication-ui/chat/src/main/res/values/azure_communication_ui_chat_strings.xml:9]: do we have any default text in Figma?

File: azure-communication-ui/chat/src/main/java/com/azure/android/communication/ui/chat/presentation/ui/chat/components/BottomBarView.kt [hunk 2]
```diff
@@ -53,6 +53,7 @@ internal fun BottomBarView(
 }
 
 private fun sendButtonOnclick(postAction: (Action) -> Unit, messageInputTextState: MutableState<String>) {
+    if (messageInputTextState.value.isBlank()) return
```
Comment (iaulakh) [inline azure-communication-ui/chat/src/main/java/com/azure/android/communication/ui/chat/presentation/ui/chat/components/BottomBarView.kt:56]: can you please check with Alex, if we need to show grey send message button when disabled?

File: azure-communication-ui/chat/src/main/java/com/azure/android/communication/ui/chat/presentation/ui/chat/components/MessageInputView.kt [hunk 3]
```diff
@@ -112,7 +110,7 @@ internal fun MessageInput(
 
                 if (textContent.isEmpty() && !focusState) {
                     BasicText(
-                        text = stringResource(R.string.azure_communication_ui_chat_enter_a_message),
```
Comment (Mohtasim) [inline azure-communication-ui/chat/src/main/java/com/azure/android/communication/ui/chat/presentation/ui/chat/components/MessageInputView.kt:115]: Do we want to use a hint here?

File: azure-communication-ui/chat/src/main/java/com/azure/android/communication/ui/chat/presentation/ui/chat/components/BottomBarView.kt (hunk 2 above)
Comment (ShaunaSong) [reply azure-communication-ui/chat/src/main/java/com/azure/android/communication/ui/chat/presentation/ui/chat/components/BottomBarView.kt]: Should have disable state.

File: azure-communication-ui/chat/src/main/res/values/azure_communication_ui_chat_strings.xml (hunk 1 above)
Comment (ShaunaSong) [reply azure-communication-ui/chat/src/main/res/values/azure_communication_ui_chat_strings.xml:9]: Updated

**Review by Mohtasim — APPROVED** [review]

**Review by iaulakh — APPROVED** [review]

**Pull request merged**